pages/contact.md

```diff
@@ -13,23 +13,24 @@ You can contact any one of the core team below for any Pa11y-related issue, this
   - If you wish to discuss an issue before taking it public
   - If you need to report a violation of [our code of conduct][code-of-conduct]
 
-You can reach us here:
+If you don't wish to contact an individual team member, you can reach us here:
```

Comment:
s/wish/want ? "If you'd prefer not to contact an individual team member..." ? 

('wish' falls foul of the sprauncy new language guidelines imo (as does 'sprauncy'))